h/views/api_config.py

```diff
@@ -0,0 +1,92 @@
+import venusian
+
+from h.util import cors
+
+
+cors_policy = cors.policy(
+    allow_headers=(
+        'Authorization',
+        'Content-Type',
+        'X-Annotator-Auth-Token',
+        'X-Client-Id',
```

Comment:
I know this is just moving code around, but it might be useful to add a short comment after this to note that it is used by the real time API.